src/Files.App/Dialogs/CreateShortcutDialog.xaml

```diff
@@ -0,0 +1,59 @@
+﻿<ContentDialog
+	x:Class="Files.App.Dialogs.CreateShortcutDialog"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:helpers="using:Files.App.Helpers"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	Title="{helpers:ResourceString Name=NewShortcutDialogTitle}"
+	DefaultButton="Primary"
+	PrimaryButtonClick="ContentDialog_PrimaryButtonClick"
+	PrimaryButtonText="{helpers:ResourceString Name=Continue}"
+	RequestedTheme="{x:Bind helpers:ThemeHelper.RootTheme}"
+	SecondaryButtonText="{helpers:ResourceString Name=Cancel}"
+	Style="{StaticResource DefaultContentDialogStyle}"
+	mc:Ignorable="d">
+
+	<Border Width="400">
+		<Grid
+			x:Name="DestinationPathGrid"
+			ColumnSpacing="8"
+			RowSpacing="8">
+			<Grid.ColumnDefinitions>
+				<ColumnDefinition Width="400" />
+				<ColumnDefinition />
+			</Grid.ColumnDefinitions>
+			<Grid.RowDefinitions>
+				<RowDefinition Height="Auto" />
+				<RowDefinition Height="Auto" />
+				<RowDefinition Height="Auto" />
+			</Grid.RowDefinitions>
+			
+			<!--  Header  -->
+			<TextBlock 
+				Margin="0, 0, 0, 20"
+				Text="{helpers:ResourceString Name=NewShortcutDialogDescription}"
+				TextWrapping="Wrap" />
+
+			<TextBlock
+				Grid.Row="1"
+				Grid.ColumnSpan="2"
+				Text="{helpers:ResourceString Name=NewShortcutDialogPrompt}" />
+
+			<!--  Path Box  -->
+			<TextBox
```

Comment:
Can you disable input?

Reply:
I don't think we should do so. This way the user can create shortcuts to files and web addresses as requested in #10187

Reply:
I may add a check on the input

Reply:
Good point, we'll need input validation then.

Reply:
I'll work on that

Reply:
It seems that using `ShellLink`s you can't create shortcut to drives. I'll try to find a solution to this